### PR TITLE
OCS4.6/4.8 Installation via UI

### PR DESCRIPTION
**Pull request by OdedViner**
Signed-off-by: Oded Viner <oviner@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/ui/views.py [hunk 1]
```diff
@@ -172,16 +197,22 @@
 }
 
 locators = {
+    "4.8": {
+        "login": login,
+        "page": page_nav,
+        "deployment": {**deployment, **deployment_4_7},
```
Comment (ebenahar) [inline ocs_ci/ocs/ui/views.py:203]: just to be sure, is `**deployment_4_7` intentional here?

Comment (OdedViner) [reply ocs_ci/ocs/ui/views.py:203]: yes. 
There is no difference between OCS4.8 and OCS4.7  installation [only MULTUS option]

**Review by ebenahar — APPROVED** [review]

File: ocs_ci/ocs/ui/views.py (hunk 1 above)
Comment (ebondare) [inline ocs_ci/ocs/ui/views.py:203]: I'd add pvc and infra dictionaries here as well, just to make sure that all the tests that can pass without any changes would pass.

Comment (OdedViner) [reply ocs_ci/ocs/ui/views.py:203]: https://github.com/red-hat-storage/ocs-ci/blob/f2aadb01fcf596a0736caf8672d267da49399fd6/ocs_ci/ocs/ui/views.py#L174
there is not conflict on this pr.. I added a new key [4.8]

Comment (ebondare) [reply ocs_ci/ocs/ui/views.py:203]: Just for anyone who might also be interested, we discussed this issue with Oded separately and he explained that pvc tests are failing for 4.8 so they require a separate effort and thus the pvc dictionary was not included in this PR. As for infra dictionary, it needs to be tested as well.

**Comment by ebondare** [comment]
LGTM

**Review by ebondare — APPROVED** [review]

**Pull request merged**